examples/lax_root_batch.py

```diff
@@ -0,0 +1,25 @@
+"""Make ROOT file of all lax output
+"""
+import hax
+from lax.lichens import sciencerun0
+
+RUN_NUMBER = 3956
+PAX_VERSION = '6.4.2'
```

Comment:
Should just make a lax-batch command if you want... I don't think people are using this though and instead just using notebook with hax then writing ROOT file at end.  Maybe could remove this?